### PR TITLE
Add EventListener Response in the Doc

### PR DESCRIPTION
**Pull request by khrm**
Add response from the EventListener in the doc. Status code and body description added.

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: docs/eventlisteners.md [hunk 1]
```diff
@@ -632,6 +632,17 @@ spec:
     value: $(body.pull_request.head.short_sha)
 ```
 
+## Response from the EventListener
```
Comment (khrm) [inline docs/eventlisteners.md:635]: Why is not getting an anchor at the top like others?
https://github.com/tektoncd/triggers/blob/445d9a434e4021118f32bcef01347c829e5daa11/docs/eventlisteners.md

File: docs/eventlisteners.md [hunk 2]
```diff
@@ -632,6 +633,17 @@ spec:
...
```
Comment (dibyom) [inline docs/eventlisteners.md:636]: Very small nit: Call this EventListener Response instead?

Comment (khrm) [reply docs/eventlisteners.md]: Done.

**Pull request merged**